SortableGridBehavior.php

```diff
@@ -65,13 +78,18 @@ public function gridSort($items)
 
     public function beforeInsert()
     {
-        /** @var ActiveRecord $model */
-        $model = $this->owner;
-        if (!$model->hasAttribute($this->sortableAttribute)) {
+        if (!$this->owner->hasAttribute($this->sortableAttribute)) {
             throw new InvalidConfigException("Invalid sortable attribute `{$this->sortableAttribute}`.");
         }
+        $this->owner->{$this->sortableAttribute} = $this->findMax() + 1;
+    }
 
-        $maxOrder = $model->find()->max($model->tableName() . '.' . $this->sortableAttribute);
```

Comment:
In principle, I have nothing to add, but only extended the current functionality and made the string into a separate method.